src/signals/incident-management/containers/Dashboard/Dashboard.tsx

```diff
@@ -1,22 +1,67 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2023 Gemeente Amsterdam
-import { useState } from 'react'
+import { useEffect, useMemo, useState } from 'react'
 
-import { AreaChart, BarChart, Filter } from './components'
-import { FilterWrapper, StyledRow } from './styled'
+import { Row } from '@amsterdam/asc-ui'
+import { capitalize } from 'lodash'
+import { Link, Route, useLocation, useHistory } from 'react-router-dom'
+
+import { Tab } from 'components/Tab'
+
+import { AreaChart, BarChart, Filter, Signaling } from './components'
+import { FilterWrapper, StyledRow, StyledTabContainer } from './styled'
+import { DASHBOARD_URL } from '../../routes'
 
 const Dashboard = () => {
   const [queryString, setQueryString] = useState<string>('')
 
+  const location = useLocation()
+
+  const history = useHistory()
+
+  const pages = useMemo(
+    () => ({
+      nu: `${DASHBOARD_URL}/nu`,
+      signalering: `${DASHBOARD_URL}/signalering`,
+    }),
+    []
+  )
+
+  useEffect(() => {
+    if (location.pathname === DASHBOARD_URL) {
+      history.push(pages.nu)
+    }
+  }, [history, location.pathname, pages])
+
   return (
     <>
       <FilterWrapper data-testid="menu">
         <Filter setQueryString={setQueryString} />
       </FilterWrapper>
       <StyledRow>
-        <BarChart queryString={queryString} />
-        <AreaChart />
+        <StyledTabContainer>
+          {Object.entries(pages).map(([name, url]) => (
+            <Tab
+              key={name}
+              className={url === location.pathname ? 'active' : ''}
+              as={Link}
+              to={url}
+            >
+              {capitalize(name)}
+            </Tab>
+          ))}
+        </StyledTabContainer>
       </StyledRow>
+      <Route path={`${DASHBOARD_URL}/nu`}>
+        <Row>
+          <BarChart queryString={queryString} />
+          <AreaChart />
+        </Row>
+      </Route>
+      <Route
+        path={`${DASHBOARD_URL}/signalering`}
+        component={Signaling}
```

Comment:
To be consistent, maybe good to remove the Row inside this component, and render it in the styled row as well.

Reply:
yeah good one!